docs/infrastructure/data_ingestion.md

```diff
@@ -0,0 +1,21 @@
+# Data Ingestion
+
+## Ingestion Pipeline
+Treeherder uses the [Celery](https://docs.celeryproject.org/en/stable/index.html) task queue software, with the RabbitMQ broker, for processing taskcluster data that is submitted to the [Pulse Guardian](https://pulseguardian.mozilla.org/) queues. It only subscribes to specific exchanges and only processes pushes and tasks for repositories that are defined in the repository.json [fixture](https://github.com/mozilla/treeherder/blob/master/treeherder/model/fixtures/repository.json). 
```

Comment:
Note to self: add a sentence about where to see the queues for a deployment (rabbitMQ dashboard) and how to see what exchanges an account is subscribed to (in pulse guardian and in the code base).